lib/s3/managed_upload.js

```diff
@@ -522,7 +532,7 @@ AWS.S3.ManagedUpload = AWS.util.inherit({
    */
   finishMultiPart: function finishMultiPart() {
     var self = this;
-    var completeParams = { MultipartUpload: { Parts: self.completeInfo } };
+    var completeParams = { MultipartUpload: { Parts: self.completeInfo.slice(1) } };
```

Comment:
We are slicing as we are following the pattern of keeping track of parts by the correctly indexed key, ie: part 1 has key value 1, and key 0 is null, so we slice it out.